trulens_eval/examples/snowflake_demo/retrieve.py

```diff
@@ -36,3 +36,38 @@ def retrieve(self, query: str):
         contents = [json.loads(t.page_content) for t in nodes]
         texts = [tc.get("text") for tc in contents]
         return texts
+
+
+class CortexSearchRetriever:
+
+    def __init__(self, limit_to_retrieve: int = 4):
+        self._limit_to_retrieve = limit_to_retrieve
+        connection_parameters = {
+            "account": os.environ["SF_ACCOUNT"],
+            "user": os.environ["SF_USER"],
+            "password": os.environ["SF_PASSWORD"],
+            "role": os.environ["SF_ROLE"],
+            "warehouse": os.environ["SF_WAREHOUSE"],
+        }
+        session = Session.builder.configs(connection_parameters).create()
+        root = Root(session)
+        self._cortex_search_service = root.databases[
+            os.environ["SF_DB_NAME"]].schemas[
+                os.environ["SF_SCHEMA"]].cortex_search_services[
+                    os.environ["SF_CORTEX_SEARCH_SERVICE"]]
+
+    def retrieve(self, query: str):
```

Comment:
Can you match the pinecone retriever's use of the context filter (or remove from both)?

Reply:
sorry, what do you mean?

Reply:
I meant that the PineconeRetriever uses `WithFeedbackFilterDocuments` (although with filter set to 0), while cortex search doesn't here. Addressing my comment here: https://github.com/truera/trulens/pull/1229